python/MooseDocs/test/extensions/tests

```diff
@@ -1,50 +1,58 @@
 [Tests]
   design = 'MooseDocs/index.md'
   issues = '#6699'
+
+  requirement = "The system shall include documentation tools that provide support for"
   [core]
     type = PythonUnitTest
     input = test_core.py
-    requirement = "The documentation system shall include basic support for a markdown-like syntax."
+    detail = "markdown-like syntax,"
   []
   [comment]
     type = PythonUnitTest
     input = test_comment.py
-    requirement = "The documentation system shall support comments within markdown syntax."
+    detail = "comments within markdown syntax,"
   []
   [acronym]
     type = PythonUnitTest
     input = test_acronym.py
-    requirement = "The documentation system shall support acronyms within markdown syntax."
+    detail = "acronyms within markdown syntax,"
     issues = '#6699 #13611'
   []
   [sqa]
     type = PythonUnitTest
     input = test_sqa.py
-    requirement = "The documentation system shall include tools for building software quality documents."
+    detail = "building software quality documents,"
     issues = '#6699 #12049'
   []
   [autolink]
     type = PythonUnitTest
     input = test_autolink.py
-    requirement = "The documentation system shall include tools for automatically cross-linking files."
+    detail = "automatically cross-linking files,"
     issues = '#6699 #13567'
   []
   [civet]
     type = PythonUnitTest
     input = test_civet.py
-    requirement = "The documentation system shall include tools for linking to continuous integration testing results."
+    detail = "linking to continuous integration testing results,"
     issues = '#12049'
   []
   [content]
     type = PythonUnitTest
     input = test_content.py
-    requirement = "The documentation system shall include tools for listing and linking to content within and across documents."
+    detail = "listing and linking to content within and across documents,"
     issues = '#9638'
   []
   [appsyntax]
     type = PythonUnitTest
     input = test_appsyntax.py
-    requirement = "The documentation system shall include tools for automatically retrieving application syntax."
+    detail = "automatically retrieving application syntax, and"
     issues = '#9638'
   []
+  [alert]
+    type = PythonUnitTest
+    input = test_alert.py
+    detail = "creating alert text boxes."
+    issues = '#15232'
+  []
 []
```

Comment:
Okay, here's what I did

Reply:
You need to add another level of nesting:

```
[Tests]
  [extensions]
  ...everything in here...
  []
[]

Reply:
If I do this, I get the following error:

```
MooseDocs/test/integrity.spec: Working Directory: /home/crswong888/projects/moose/python/MooseDocs/test/integrity
MooseDocs/test/integrity.spec: Running command: python -m unittest -v test_spec_files
MooseDocs/test/integrity.spec: testSpec (test_spec_files.TestSpecFiles) ... /opt/moose/miniconda/lib/python3.7/site-packages/pybtex/database/__init__.py:27: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
MooseDocs/test/integrity.spec:   from collections import Mapping
MooseDocs/test/integrity.spec: /opt/moose/miniconda/lib/python3.7/site-packages/pybtex/utils.py:27: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
MooseDocs/test/integrity.spec:   from collections import (
MooseDocs/test/integrity.spec: FAIL
MooseDocs/test/integrity.spec: 
MooseDocs/test/integrity.spec: ======================================================================
MooseDocs/test/integrity.spec: FAIL: testSpec (test_spec_files.TestSpecFiles)
```

Is it just my machine? I still use the old school MOOSE package.

Reply:
You still don't have the nesting correct.

```
[Tests]
  [extensions]
    requirement = "The system shall include documentation tools that provide support for"
    [core]
      type = PythonUnitTest
      input = test_core.py
      detail = "markdown-like syntax,"
    []
    [comment]
      type = PythonUnitTest
      input = test_comment.py
      requirement = "The documentation system shall support comments within markdown syntax."
      detail = "comments within markdown syntax,"
    []
    # all the other blocks here...
  []
[]
```

Reply:
@aeslaughter did you test out this format? I found that it doesn't work...